src/test/kotlin/stringCalculator/CalculatorTest.kt

```diff
@@ -0,0 +1,36 @@
+package stringCalculator
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import org.junit.jupiter.params.provider.ValueSource
+
+internal class CalculatorTest {
+
+    @ParameterizedTest
+    @MethodSource("validTestcase")
+    fun `calculate 에 유효한 값을 입력하면 사칙연산을 한다`(input: String, expected: Int) {
```

Comment:
```suggestion
    fun `수식이 입력되면 사칙연산을 한다`(input: String, expected: Int) {
```
함수명을 좀 더 간결하게 작성하면 좋을 것 같아요.